Packaging/Package.appxmanifest

```diff
@@ -301,6 +301,45 @@
         </uap:Extension>
       </Extensions>
     </Application>
+    <Application Id="DevHome.HostsFileEditor" Executable="DevHome.HostsFileEditor\DevHome.HostsFileEditor.exe" EntryPoint="Windows.FullTrustApplication">
+      <uap:VisualElements DisplayName="ms-resource:HostsFileEditorAppDisplayName" Description="ms-resource:HostsFileEditorAppDescription" BackgroundColor="transparent" Square150x150Logo="Assets\HostsFileEditor\Square150x150Logo.png" Square44x44Logo="Assets\HostsFileEditor\Square44x44Logo.png">

```

Comment:
Don't we want these in Windows Tools rather than the top-level start menu? Hypothetically, think about the customers that would prefer not having these tools populate their start menu. 

One of the downsides of a MAP is that if they just wanted to get rid of "Host File Editor" for whatever reason, DevHome would be uninstalled with it unintentionally (albeit this is a start menu UX gap that we could address), and they wouldn't even know that just from looking at Start.

Reply:
There's no public way to put these in "Windows Tools" page. But I can put them in a "DevHome Utilities" folder in All Apps. @EricJohnson327, @krschau, @cinnamon-msft, thoughts?

Reply:
Discussed offline. Marking applist entries none.